Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rust:1.39 as builder
```

Comment:
is there a way we can use latest rust stable?

Reply:
of cors